[TW-1225] Split importing and exporting (#4944)
postmanlabs/postman-docs: src/pages/docs/getting-started/importing-and-exporting/exporting-data.md

---
title: 'Exporting data from Postman'
updated: 2023-07-19
contextual_links:
  - type: section
    name: "Additional resources"
  - type: subtitle
    name: "Videos"
  - type: link
    name: "Import and Export Data in Postman"
    url: "https://youtu.be/KdaiVdNMgL4"
---

You can export your Postman data, including collections, environments, global variables, and data dumps, as JSON files. You can import these files back into any Postman instance, share them with others, or use them with [Newman](/docs/collections/using-newman-cli/command-line-integration-with-newman/), Postman's command-line collection runner.

## Contents

* [Exporting collections](#exporting-collections)
* [Exporting environments](#exporting-environments)
* [Exporting data dumps](#exporting-data-dumps)

## Exporting collections

1. Select the more actions icon <img alt="More actions icon" src="https://assets.postman.com/postman-docs/icon-more-actions-v9.jpg#icon" width="16px"> next to the collection, then select **Export**.

   <img alt="Export collection" src="https://assets.postman.com/postman-docs/export-collection-v9.1.jpg" width="350px" />

> 1. Select the format you'd like your collection to export as.

Reviewer: ```suggestion
1. Select the export format for your collection.
```
